### PR TITLE
Add the profile component

### PR DESCRIPTION
**Pull request by ethanjc**
This PR add the Profile component that supports an image and a full name. It automatically falls back to the user's initials with a generated gradient if the image does not exist or does not load properly.

https://user-images.githubusercontent.com/20915535/185000106-05bd80ea-76eb-4321-9356-e533eece4486.mp4

### CODE REVIEW

File: packages/constellation/package.json [hunk 1]
```diff
@@ -150,6 +150,8 @@
     "@radix-ui/react-toggle-group": "0.1.3",
     "@radix-ui/react-tooltip": "1.0.0",
     "@tanstack/react-table": "8.5.11",
+    "@types/randomcolor": "^0.5.6",
+    "randomcolor": "^0.6.2",
```
Comment (blockdylanb) [inline packages/constellation/package.json]: @ethanjc mind pinning the version?

Comment (ethanjc) [reply packages/constellation/package.json]: Done!

File: packages/constellation/src/components/Primitives/Profile/Fallback.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    const secondColor = randomColor({
+      format: 'hex',
+      luminosity: 'light',
+      seed: fullName[fullName.length - 1],
+    })
+
+    const gradient = `linear-gradient(${angle}deg, ${firstColor} ${firstPercent}%, ${secondColor} ${secondPercent}%)`
+
+    return { gradient, initials }
+  }, [name])
```
Comment (noahub) [inline packages/constellation/src/components/Primitives/Profile/Fallback.tsx]: Could this be split out into a util and/or custom hook?

**Review by noahub — DISMISSED** [review]
I had a small comment, but I think that could be done in a follow-up PR if you want to get this one merged.

**Review by noahub — APPROVED** [review]

**Pull request merged**